sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Utilities/MetricsAnalyzerBase.cs

```diff
@@ -65,6 +65,7 @@ protected sealed override MetricsInfo GetMessage(SyntaxTree syntaxTree, Semantic
                 FunctionCount = metrics.FunctionCount,
                 PublicApiCount = metrics.PublicApiCount,
                 PublicUndocumentedApiCount = metrics.PublicUndocumentedApiCount,
+                ExecutableLines = metrics.GetExecutableLinesCount(),

```

Comment:
All other metrics are properties, this one is method, if possible I would change this.